src/ishikawa_backend.erl

```diff
@@ -47,7 +47,11 @@
 
 -record(state, {actor :: actor(),
                 vv :: timestamp(),
-                members :: [term()]}).
+                members :: [term()],
+                svv :: timestamp(),
+                rtm :: timestamp_matrix(),
+                to_be_delivered_queue :: [{timestamp(), term()}],
+                to_be_ack_queue :: [{term(), [term()]}]}).
```

Comment:
Can you be more specific about the types here?  It's hard to tell what this is a list of...